pages/community/community-support-precice.md

```diff
@@ -0,0 +1,52 @@
+
+## Funding and support options
+
+There are three different ways to fund and support the development of preCICE.
+
+### 1. Research project partner
+
+Include preCICE as a project partner in your research project proposal or team up with the preCICE developers to write a proposal. This is the best option when you want to benefit from the research and scientific expertise of the preCICE team, and increase the chances that your project is funded. The developers of preCICE are experts in state-of-the-art numerical methods, high-performance computing, and software engineering for multi-X coupled problems. Reviewers appreciate the highly recognized expertise of preCICE, demonstrated by our fast growing [number of citations](https://scholar.google.com/scholar?hl=en&cites=5053469347483527186), and [vibrant community](community-projects.html).
+
+Directly [contact Jun. Prof. Benjamin Uekermann](https://www.ipvs.uni-stuttgart.de/departments/us3/) to discuss a collaboration.
+
+### 2. Support license
+
+A support license is the easiest way to support preCICE. By acquiring a support license, you directly contribute to the development of preCICE through a fixed amount per year. In exchange, you receive a number of benefits, in particular timely and direct support from the preCICE developers around any issues that may arise when using preCICE. We provide two service levels, with discounted rates for academia.
+
+#### Direct cost in a research project
+
+As an academic user you can declare the support license, like any software license, as a direct cost/consumable in your research proposal (see e.g. section 2.1 in [basic module DFG guidelines](https://www.dfg.de/formulare/52_01/52_01_en.pdf)). It is also recommended to include the support license as a risk mitigation strategy to highlight that you, as a project lead, have the support for a successful implementation. In addition, as part of the support license, you can leverage the preCICE website and community for your communication/dissemination activities.
+
+[Speak to Jun. Prof. Benjamin Uekermann](https://www.ipvs.uni-stuttgart.de/departments/us3/) to learn more.
+
+#### Benefits
+
+|   | Basic support license <br>(12 months, billed anually)  | Extended support license <br>(12 months, billed anually)  |
+| - | - | -|
+| **discounts for preCICE Workshop** | up to 3 tickets | up to 3 tickets |
+| **priority support**   | via Discourse, up to 3 badges | via Discourse, up to 3 badges  |
+| **visibility/dissemination**   | listed as supporter, project featured on website | listed as supporter, project featured on website  |
+| **private support**   |  | via email or Zoom |
+| **on-site support**    |  | up to 2 days of training or support |
+| **service level agreement** | | upon request |
```

Comment:
What is a "service level agreement"? An actual contract? If this is not provided for the standard level, what do they get instead?

Reply:
Yes, that is correct, a contract that clarifies exactly what the levels of support are, how support is provided, typically with metrics (not applicable here but think of something like uptime), when and how the metrics are reviewed, by who (reporting structures), and what happens if something goes wrong etc.
If you buy software/service for everyday use, you don't get this level of support. Which is one of the reasons 'business software' is an order of magnitude more expensive than 'consumer software' even though the features of the software are very similar/the same.
So back to our case the standard level gets support as specified in the table but no guarantee, that say, somebody responds within a certain time frame.

Reply:
> So back to our case the standard level gets support as specified in the table but no guarantee, that say, somebody responds within a certain time frame.

Ah, I see. This was not completely clear to me, so maybe it needs some further clarification. From the table I expected that this is the same for both levels. I guess for companies this is clear, but if we want to differentiate it, we could clrify it.

Reply:
Yes, if you need an SLA from your providers you should know 😆. Let's hope we don't have to provide one for now.